src/main/java/uk/gov/hmcts/befta/util/MapVerifier.java

```diff
@@ -122,15 +121,15 @@ private MapVerificationResult verifyMap(String fieldPrefix, Map<String, Object>
                         //
                     } else if (expectedValue == null) {
                         badValueMessages.add("Must be null: " + commonKey);
-                    } else if (actualValue == null) {
+                    } else if (actualValue == null && !nullable((String) expectedValue)) {
```

Comment:
else if (actualValue == null && !nullable((String) expectedValue))
--> else if (actualValue == null && isNoneNullablePlaceholder((String) expectedValue))

and 

private boolean isNoneNullablePlaceholder(String strExpectedValue) {
        ExpectedValuePlaceholder expectedValuePlaceholder = ExpectedValuePlaceholder.getByValue(strExpectedValue);
        return expectedValuePlaceholder != null &&  !expectedValuePlaceholder.isNullable();
}

Reply:
changed method name